tests/e2e/catalog/items/test_async_item.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from mpt_api_client.exceptions import MPTAPIError
+from mpt_api_client.rql.query_builder import RQLQuery
+
+pytestmark = [pytest.mark.flaky]
+
+
+@pytest.fixture
+async def async_created_item(logger, async_mpt_vendor, item_data):
+    service = async_mpt_vendor.catalog.items
+    item = await service.create(item_data)
+    yield item
+    try:
+        await service.delete(item.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete item {item.id}: {error.title}")  # noqa: WPS421
+
+
+def test_create_item(async_created_item):
+    assert async_created_item.name == "e2e - please delete"
+
+
+async def test_update_item(async_mpt_vendor, async_created_item):
+    service = async_mpt_vendor.catalog.items
+    update_data = {"name": "e2e - delete me (updated)"}
+    item = await service.update(async_created_item.id, update_data)
+    assert item.name == "e2e - delete me (updated)"
+
+
+@pytest.mark.skip(reason="Leaves test items in the catalog")
+async def test_review_and_publish_item(async_mpt_vendor, async_mpt_ops, async_created_item):
+    item = await async_mpt_vendor.catalog.items.review(async_created_item.id)
+    assert item.status == "Pending"
+
+    item = await async_mpt_ops.catalog.items.publish(async_created_item.id)
+    assert item.status == "Published"
+
+
+async def test_get_item(async_mpt_vendor, item_id):
+    if item_id is None:
+        pytest.skip("No item ID configured for testing")
```

Comment:
If I understand correctly, we are skipping this test because of a missing piece of config. Why would we want such a scenario? I think we should rather fail this test because if it gets skipped, it undermines the consistency of the whole suite. In other words, if this test normally fails you can make the whole suite pass just by missing a piece of config